<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>armemus</class>
 <widget class="QMainWindow" name="armemus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="mainWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="mainLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="outputWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="outputLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionStep"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent &amp;Files</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRecent_P_rojects">
     <property name="title">
      <string>Recent P&amp;rojects</string>
     </property>
    </widget>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="menuRecent_P_rojects"/>
    <addaction name="separator"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionCloseFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionStep"/>
    <addaction name="actionBuildOptions"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/step.png</normaloff>:/icons/step.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/build.png</normaloff>:/icons/build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionBuildOptions">
   <property name="text">
    <string>Build Options</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCloseFile">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>...</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
